lib/context_adapter.js

```diff
@@ -126,6 +129,20 @@ function startup(callback) {
             op: caConfig.OPERATION_TYPE.SERVER_START
           },
           'Server started at %s...', caServer.hapiServer.info.uri);
+
+        if (enableProofOfLifeLogging) {
+          proofOfLifeInterval = setInterval(function () {
```

Comment:
Fixed in https://github.com/telefonicaid/context-adapter/commit/d11b4d20d258924b223803aebc7c5ff237ac5277